### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by philipp94831**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -8,16 +8,20 @@ variables:
...
   - group: sign
   - group: ossrh
+  - group: changelog
 
 resources:
   repositories:
     - repository: templates
       type: github
       name: bakdata/bakdata-project-templates
+      ref: tmp/upload-snapshot
```
Comment (JakobEdding) [inline azure-pipelines.yml:18]: TODO?

Comment (philipp94831) [reply azure-pipelines.yml]: Fixed

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```
Comment (JakobEdding) [inline gradle/wrapper/gradle-wrapper.properties:3]: Doesn't the binary and .bat file change as well from 7.1 to 7.6?

Comment (philipp94831) [reply gradle/wrapper/gradle-wrapper.properties:3]: I don't know when this changes. There is a command to change wrapper version and I don't do that manually

**Review by JakobEdding — APPROVED** [review]

**Pull request merged**